torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -63,6 +63,44 @@ bool isSupported(Node* node) {
     case aten::cos:
     case aten::sin:
     case aten::tan:
+    case aten::acos:
+    case aten::asin:
+    case aten::atan:
+    case aten::cosh:
+    case aten::sinh:
+    case aten::tanh:
+    case aten::abs:
+    case aten::sqrt:
+    case aten::rsqrt:
+    case aten::floor:
+    case aten::ceil:
+    case aten::round:
+    case aten::trunc:
+      return true;
+    default:
+      return false;
+  }
+}
+
+bool isInterpretedOnly(Node* node) {
```

Comment:
Instead of adding this extra function, just #ifdef out the part of the main list that isn't supported by llvm.

Reply:
This makes sense. Resolved, but let me know if I missed anything